src/DotNetWorker.Grpc/AppContextUtils.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.Functions.Worker.Grpc
+{
+    internal class AppContextUtils
+    {
+        /// <summary>
+        /// Returns a boolean value indicating whether the worker is running with native host(placeholder support).
```

Comment:
Running native and placeholder mode are different and although our setup will run in native in placeholder mode only, that is not a requirement. I think you can drop the placeholder comment.

Suggest something like this:

```
 /// <summary>
 /// Indicates whether the worker is running in the native host.
 /// </summary>
 /// <returns><see langword="true"/> if the worker is running in the native host; otherwise,<see langword="false"/>.</returns>
```

Reply:
Removed this helper.